vocabularies/georesources-report-types.ttl

```diff
@@ -0,0 +1,538 @@
+    skos:prefLabel "Presentation"@en ;
+    skos:topConceptOf grt:conceptScheme .
+
+grt:departmental-reports a skos:Collection ;
+    skos:definition "Non-Tenure reports which were produced by Queensland's Department of Natural Resources, Mines and Energy."@en ;
+    skos:member grt:annual-report,
+        grt:any-other-report,
+        grt:commissioned-industry-study-or-report,
+        grt:departmental-drilling,
+        grt:geological-survey-of-queensland-publication,
+        grt:geological-survey-of-queensland-record,
+        grt:industry-consultative-report,
+        grt:presentation,
+        grt:queensland-government-mining-journal,
+        grt:soils--26-land-resources ;
+    skos:prefLabel "Departmental Reports"@en .
+
+grt:exploration-grants a skos:Collection ;
+    skos:definition "Reports which were funded either fully or partially by a Queensland Government grant in aid of exploration."@en ;
+    skos:member grt:collaborative-drilling-initiative---final--28completed-29,
+        grt:collaborative-drilling-initiative---final-report,
+        grt:collaborative-drilling-initiative---proposals,
+        grt:industry-network-initiative---final--28completed-29 ;
+    skos:prefLabel "Exploration Grants"@en .
+
+grt:external-reports a skos:Collection ;
+    skos:definition "Non-Tenure reports which were produced by bodies external to Queensland's Department of Natural Resources, Mines and Energy."@en ;
+    skos:member grt:any-other-report,
+        grt:article,
+        grt:presentation,
+        grt:technical-report,
+        grt:thesis ;
+    skos:prefLabel "External Reports"@en .
+
+grt:geophysics-reports a skos:Collection ;
+    skos:definition "Reports generated during the collection, processing or interpretation of geophysics."@en ;
+    skos:member grt:acquisition-report,
+        grt:logistics-report,
+        grt:seismic-reprocessing-report,
+        grt:seismic-survey-final-report,
+        grt:seismic-survey-report-other ;
+    skos:prefLabel "Geophysics Reports"@en .
+
+grt:geothermal-reports a skos:Collection ;
+    skos:definition "Reports pertaining to Geothermal tenures."@en ;
+    skos:member grt:geothermal-annual-reserves-reports,
+        grt:geothermal-injection-reports,
+        grt:geothermal-injection-testing-reports,
+        grt:geothermal-production-reports,
+        grt:geothermal-production-testing ;
+    skos:prefLabel "Geothermal Reports"@en .
+
+grt:greenhouse-gas-reports a skos:Collection ;
+    skos:definition "Reports pertaining to Greenhouse Gas Storage tenures."@en ;
+    skos:member grt:greenhouse-gas-injection-report,
+        grt:greenhouse-gas-storage-injection-report,
+        grt:greenhouse-gas-stream-storage-capacity-report ;
+    skos:prefLabel "Greenhouse Gas Reports"@en .
+
+grt:mine-plans a skos:Collection ;
+    skos:definition "Mine plans."@en ;
+    skos:member grt:mine-plan-lodgement ;
+    skos:prefLabel "Mine Plans"@en .
+
+grt:non-current-reports a skos:Collection ;

```

Comment:
I would remove this collection.

Instead:
a. If the concept has been superceded, then add it as an altLabel or even a hiddenLabel in the concept that supercedes it. Delete the concept.  
b. If the concept has been discontinued, put "This report has been discontinued." in the skos:definition for each of the discontinued concepts. Also add a skos:note "Deprecated". This will allow us to write a SPARQL query that excludes the deprecated terms from a drop-down list.